Formula/g/gitmux.rb

```diff
@@ -0,0 +1,24 @@
+class Gitmux < Formula
+  desc "Git status in tmux status bar"
+  homepage "https://github.com/arl/gitmux"
+  url "https://github.com/arl/gitmux/archive/refs/tags/v0.11.2.tar.gz"
+  sha256 "c62b180415c272743d01531b911091b9c35911be4ec4aae3e7bfceddf5094f6c"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-s -w -X main.version=#{version}"
+
+    system "go", "build", *std_go_args(ldflags:)
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/gitmux -V")
+
+    system "git", "init", "--initial-branch=gitmux"
+
+    ENV.remove "PATH", HOMEBREW_SHIMS_PATH
```

Comment:
Why is this needed? It seems like a bug

Reply:
@SMillerDev under the hood `gitmux -dbg` executes `git status --porcelain=v1 --branch -z`. With git shim I get this error:

```shell
error: exec /opt/homebrew/Library/Homebrew/shims/shared/git 'status --porcelain=v1 --branch -z': exit status 1
```

so maybe there's an issue with a shim, since plain git works fine. This is for the test only though, so risk seems low.

Reply:
What happens if you add
```ruby
shell_output("git status --porcelain=v1 --branch -z")
```
to the test block before modifying `PATH`?

Reply:
@carlocab there's no issue in plain git execution, but how gitmux passes args to git, which the shim can't handle unlike plain git.

Reply:
@chenrui333 @SMillerDev @carlocab, is there anything I can do to move this forward?

Reply:
What about just add git formula for test? Not removing, but adding.
```suggestion
    ENV.prepend_path "PATH", Formula["git"].opt_bin
```

Reply:
@daeho-ro that's good idea. However, I'm thinking that git comes with cli tools, so maybe `uses_from_macos "git"` would be more appropriate? what would the path be in that case?

Reply:
@daeho-ro unfortunately prepend didn't work, I get the same error:

```plain
error: exec /opt/homebrew/Library/Homebrew/shims/shared/git 'status --porcelain=v1 --branch -z': exit status 1
```

Reply:
Prepend should work only if this depends on "git" not uses_from_macos.

Reply:
Yes, I know, I tried `ENV.prepend_path "PATH", "/usr/bin"`. Is there a reason for it to not work?

Reply:
hey @daeho-ro, please see above. Only removing the shim allowed the test to pass.